### PR TITLE
[IRRC-62] Refactor Screening Table to use Vuex

### PR DESCRIPTION
**Pull request by lauratsui**
Added getter in vuex to filter screening students and refactored screening table.
Also made some abstraction changes to the table, as Vue discouraged using v-if and v-for together.

### CODE REVIEW

File: frontend/src/store.js [hunk 1]
```diff
@@ -31,6 +31,13 @@ export default new Vuex.Store({
   },
   getters: {
     students: (state) => state.students,
-    ...nativeLanguageGetters
-  },
+    ...nativeLanguageGetters,
+    screeningStudents: (state) => state.students.filter((student) => {
+      if (student.status.Description == "SCREENING") {
```
Comment (aelbore) [inline frontend/src/store.js]: @lauratsui can you try this?

```javascript
 state.students.filter((student) => student.status.Description === "SCREENING")
```

**Review by aelbore — CHANGES_REQUESTED** [review]

File: frontend/src/store.js (hunk 1 above)
Comment (aelbore) [reply frontend/src/store.js]: it's shortcut for `if else`

Comment (lauratsui) [reply frontend/src/store.js]: Thanks, made the changes in the latest 5th commit

**Review by aelbore — APPROVED** [review]
LGTM

**Review by dzldebby — APPROVED** [review]
LGTM!

**Pull request merged**